fda-model/alternatives/Alternative.cs

```diff
@@ -145,34 +140,33 @@ public AlternativeResults AnnualizationCompute(interfaces.IProvideRandomNumbers
-                        double aaeqDamage = ComputeEEAD(eadSampledBaseYear, eadSampledFutureYear);
+                        double aaeqDamage = ComputeEEAD(eadSampledBaseYear, baseYear, eadSampledFutureYear, futureYear, periodOfAnalysis, discountRate);
                         aaeqResult.AddConsequenceRealization(aaeqDamage, i);
                     }
                     aaeqResult.ConsequenceHistogram.ForceDeQueue();
-                    aaeqResults.AddConsequenceResult(aaeqResult);
+                    alternativeResults.AddConsequenceResults(aaeqResult);
                 }
-                alternativeResults.AddConsequenceResults(aaeqResults);
             }
             return alternativeResults;
         }
         //TODO: these functions should be private, but currently have unit tests 
         //so these will remain public until the unit tests are re-written on the above public method
-        public double ComputeEEAD(double baseYearEAD, double mostLikelyFutureEAD){
+        public static double ComputeEEAD(double baseYearEAD, int baseYear, double mostLikelyFutureEAD, int mostLikelyFutureYear, int periodOfAnalysis, double discountRate){
 
             //probably instantiate a rng to seed each impact area differently
 
-            double[] interpolatedEADs = Interpolate(baseYearEAD, mostLikelyFutureEAD, _currentYear.Year, _futureYear.Year, _periodOfAnalysis);
-            double sumPresentValueEAD = PresentValueCompute(interpolatedEADs, _discountRate);
-            double averageAnnualEquivalentDamage = IntoAverageAnnualEquivalentTerms(sumPresentValueEAD, _periodOfAnalysis, _discountRate);
+            double[] interpolatedEADs = Interpolate(baseYearEAD, mostLikelyFutureEAD, baseYear, mostLikelyFutureYear, periodOfAnalysis);
+            double sumPresentValueEAD = PresentValueCompute(interpolatedEADs, discountRate);
+            double averageAnnualEquivalentDamage = IntoAverageAnnualEquivalentTerms(sumPresentValueEAD, periodOfAnalysis, discountRate);
             return averageAnnualEquivalentDamage;
         }
-        private double IntoAverageAnnualEquivalentTerms(double sumPresentValueEAD, Int64 periodOfAnalysis, double discountRate)
+        private static double IntoAverageAnnualEquivalentTerms(double sumPresentValueEAD, Int64 periodOfAnalysis, double discountRate)
         {
             double presentValueInterestFactorOfAnnuity = (1 - (1 / Math.Pow(1 + discountRate, periodOfAnalysis))) / discountRate;
             double averageAnnualEquivalentDamage = sumPresentValueEAD / presentValueInterestFactorOfAnnuity;
             return averageAnnualEquivalentDamage;
         }
-        private double PresentValueCompute(double[] interpolatedEADs, double discountRate)
+        private static double PresentValueCompute(double[] interpolatedEADs, double discountRate)
         {
             Int64 periodOfAnalysis = interpolatedEADs.Length;
```

Comment:
Another use of int64 where it is probably not necessary. The default int object should always be "int" unless there is a good reason.
The first answer in this post has some good info:
https://stackoverflow.com/questions/9696660/what-is-the-difference-between-int-int16-int32-and-int64

Reply:
thanks for pointing this out. I changed all int64 to int. It seems like there should not be any integer in our software that reaches billions. It is possible that results are in the billions, but all that stuff is tracked as doubles.